### PR TITLE
Module 2

### PR DESCRIPTION
**Pull request by avdhut1977**
Module 2 Project

### CODE REVIEW

File: todo_app/app.py [hunk 1]
```diff
@@ -1,23 +1,205 @@
...
+token = Config.TRELLO_TOKEN
+
+headers = {
+   "Accept": "application/json"
+}
+
+def get_tasks(s_id):
+    url = f"https://api.trello.com/1/lists/{s_id}/cards"
+    queryparams = {"key": key, "token": token}
+    jsonResponse = requests.request("GET", url, params=queryparams).json()
```
Comment (Jonesey13) [inline todo_app/app.py]: In python variable names should be snake_case

File: todo_app/app.py [hunk 2]
```diff
@@ -1,23 +1,205 @@
...
+    data = [] 
+    for jsonRecord in jsonResponse:
+        data.append({'id': jsonRecord['id'], 'name': jsonRecord['name']}) 
+    return data
+
+def get_task(id):
+    url = f"https://api.trello.com/1/cards/{id}"
+    queryparams = {"key": key, "token": token}
+    jsonResponse = requests.request("GET", url, params=queryparams).json()  
+    task = {'id': jsonResponse['id'], 'name': jsonResponse['name'], 'status_id': jsonResponse['idList'], 'board_id':jsonResponse['idBoard'] }
```
Comment (Jonesey13) [inline todo_app/app.py]: There should be a python class for handling task objects (with a constructor)

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
This is great but you're missing a few important requirements:

- Your `README.md` should explain how to add the trello credentials
- The project currently doesn't run as described as requests is missing (please run `poetry add requests`
- There is no class for todo tasks

**Comment by avdhut1977** [comment]
Review changes have ben incorporated

File: README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ You'll also need to clone a new `.env` file from the `.env.tempalate` to store l
 
 ```bash
 $ cp .env.template .env  # (first time only)
+Trello Key and Token has been set in .env to use Trello APIs
```
Comment (hugh-emerson) [inline README.md:31]: You actually mean TRELLO_KEY and TRELLO_TOKEN.

Also you give no explanation for what they are.

More minor point - this shouldn't be inside the ` ```bash` section. You can preview what the document will look like in VSCode or you can see it when you view the repository in GitHub.

File: pyproject.toml [hunk 4]
```diff
@@ -8,6 +8,11 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+requests = "^2.24.0"
+pytest = "^6.0.1"
+python-dateutil = "^2.8.1"
+selenium = "^3.141.0"
+pytest-cov = "^2.10.1"
```
Comment (hugh-emerson) [inline pyproject.toml:15]: These didn't need to be added yet

**Review by hugh-emerson — APPROVED** [review]
You've addressed Alex's comments 👍 I've left some minor points but you can merge when you're ready.

**Pull request merged**